### PR TITLE
studio: quit to dashboard for mt (breaks on single tenant)

### PR DESCRIPTION
**Pull request by achen5671**
## Summary
https://tsu.atlassian.net/browse/IR-2621

## References
closes #_insert number here_

### CODE REVIEW

File: packages/editor/src/components/projects/ProjectsPage.tsx [hunk 1]
```diff
@@ -212,7 +213,11 @@ const ProjectsPage = ({ studioPath }: { studioPath: string }) => {
     }
   })
 
-  const installedProjects = projectFindQuery.data.filter(() => projectCategoryFilter.installed.value)
+  const isOwner =
+    projectPermissionsFindQuery.data?.find((permission) => permission.userId === selfUser.id.value)?.type === 'owner'
```
Comment (DanielBelmes) [inline packages/editor/src/components/projects/ProjectsPage.tsx:217]: I'm assuming we also want to show projects that you have `editor` permissions on. I am unsure what to do about `reviewer` permissions

**Review by HexaField — CHANGES_REQUESTED** [review]
This solution breaks the base engine. it must be implemented in the MT project instead

**Comment by HexaField** [comment]
cc @hanzlamateen my proposed solution for this is here https://github.com/theinfinitereality/eepro-multitenancy/pull/250

**Comment by DanielBelmes** [comment]
@HexaField ah I was told this was for single tenant as well. That in single tenant we wanted to hide projects you don't have permissions for. From reading the ticket I think this is for MT only and we should close the PR.

**Comment by hanzlamateen** [comment]
> cc @hanzlamateen my proposed solution for this is here https://github.com/theinfinitereality/eepro-multitenancy/pull/250

Sounds good. I just requested some changes in that PR

**Pull request closed without merging**